scripts/trigger-build-helm-charts.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 
 vcs=github
 org=lightbend
-project=console-gui
+project=build-helm-charts
```

Comment:
I wouldn't mind if you hard-wired that in the curl url again.  Or called the variable something else.  But leaving this as is works too.

Reply:
Yeah, both work for me too. I left it like this because it looks easier to me than mixes with other CircleCI+curl parameters.